src/org/ods/services/JenkinsService.groovy

```diff
@@ -23,10 +23,9 @@ class JenkinsService {
         }
 
         logger.debug "Collecting (applicable) testresults from location: '${xUnitResultDir}'"
-        def logCommand = logger.debugMode ? '' : 'set +x'
         script.sh(
             script: """
-                ${logCommand}
+                ${logger.shellScriptDebugFlag}
```

Comment:
Hmm, this is a bit confusing for me. Everywhere else, debugging is opt-in, whereas here it is opt-out. Not all places use `${logger.shellScriptDebugFlag}`, so most have `-x` (the Jenkins default), and only the few places here set `+x` when debugging is off.

I guess much more effort would be needed to have this consistent everywhere (e.g. by making `PipelineSteps` aware of this and modifying the passed `script`).

I'd probably leave this like it is unless we go all-in (which I would not do right now).

Reply:
@michaelsauter  confused... what do you want, or not :)?